client/src/@types/game.d.ts

```diff
@@ -0,0 +1,14 @@
+import { Player } from './player';
+import { Venue } from './venue';
+
+export interface Game {
+  id: string;
```

Comment:
Are you mapping _id to id somewhere or how does this work? The API call will return _id.